tutorials/getting_started/rules_basic.md

```diff
@@ -51,8 +51,8 @@ This brings up a form that lets you choose the type of the trigger and enter som
 
 ![rule triggers dialog](images/rule-triggers-dialog.png)
 
-The title and description are optional and will be automatically populated with meaningful information when you select the trigger.
-However pay attention and if there is more information that might be useful to you later add that.
+~~The title and description are optional and will be automatically populated with meaningful information when you select the trigger.
+However pay attention and if there is more information that might be useful to you later add that.~~
 
```

Comment:
@rkoshak I made a change here just to remember that this no longer matches how MainUI looks. You can no longer enter title and description at this stage of the "wizard", so it might be confusing to the user when what's shown doesn't match what they see.